SampleApp/Program.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Text.Json;
+
+namespace SampleApp
+{
+    public class Program
+    {
+        /// <summary>
+        /// Session key for the cached forecast data
+        /// </summary>
+        private const string ForecastCacheKey = "forecast";
+
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+            builder.Services.AddAuthorization();
+
+            // Add session state support
+            builder.Services.AddSession(options =>
+            {
+                options.IdleTimeout = TimeSpan.FromSeconds(30);
+                options.Cookie.IsEssential = true;
+            });
+
+            // Add DynamoDB distributed cache for sessions
+            builder.Services.AddAWSDynamoDBDistributedCache(options =>
+            {
+                options.TableName = "weather-sessions-cache";
+                options.CreateTableIfNotExists = true;
+                options.UseConsistentReads = true;
+                options.PartitionKeyName = "id";
+                options.TTLAttributeName = "cache_ttl";
+            });
+
+            // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+            builder.Services.AddEndpointsApiExplorer();
+            builder.Services.AddSwaggerGen();
+
+            var app = builder.Build();
+
+            // Configure the HTTP request pipeline.
+            if (app.Environment.IsDevelopment())
+            {
+                app.UseSwagger();
+                app.UseSwaggerUI();
+            }
+
+            app.UseHttpsRedirection();
+
+            app.UseAuthorization();
+
+            app.UseSession();
+
+            var summaries = new[]
+            {
+                "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+            };
+
+            app.MapGet("/weatherforecast", (HttpContext httpContext) =>
```

Comment:
Can you also add a route for "/" that just returns back some text like welcome to the DDB cache sample. I don't want somebody getting a 404 just after launching the app.